scrape_rec/spiders/base_realestate.py

```diff
@@ -65,4 +69,4 @@ def parse(self, response):
             yield response.follow(link, callback=self.process_link)
 
         next_link = response.xpath(self.next_link_xpath).extract_first()
-        yield response.follow(next_link)
\ No newline at end of file
+        yield response.follow(next_link, dont_filter=True)
```

Comment:
What's the meaning of `dont_filter` here?

Reply:
No longer relevant but scrapy has a default duplicate filter that doesn't let you go to the same url twice. In our case, the idea was to use a redis filter but checking postgres pk is better